nix/modules/devShells.nix

```
@@ -28,6 +28,11 @@
           '';
         };
 
+        holochainBinaries = pkgs.mkShell {
+          inputsFrom = [ self'.devShells.rustDev ];
+          packages = [ self'.packages.holochain self'.packages.lair-keystore ];
+        };
```

without this it's not obvious when the shell is loaded

```suggestion
          shellHook = ''
            export PS1='\n\[\033[1;34m\][holochainBinaries:\w]\$\[\033[0m\] '
          '';
        };
```